services/QuillLMS/client/app/bundles/Staff/helpers/comprehension.tsx

```diff
@@ -53,13 +54,20 @@ export const getPromptsIcons = (activityData, promptIds: number[]) => {
   return {};
 }
 
-export const getUniversalIcon = (universal: boolean) => {
-  if(universal) {
+export const getCheckIcon = (value: boolean) => {
+  if(value) {
     return (<img alt="quill-circle-checkmark" src={quillCheckmark} />)
   } else {
-    return (<div />);
+    return (<img alt="quill-circle-checkmark" src={quillX} />);
   }
 }
+// export const getXIcon = (value: boolean) => {
```

Comment:
Commented out code. It's probably worth just doing a sweep for code that is commented out in this PR.